rotkehlchen/db/l2withl1feestx.py

```diff
@@ -17,6 +22,7 @@
 
 
 class DBL2WithL1FeesTx(DBEvmTx):
+    AUTHORIZATION_DATA_START_INDEX = 14
```

Comment:
same here. Mark it as classvar.
```suggestion
    AUTHORIZATION_DATA_START_INDEX: ClassVar[int] = 14
```